universum/modules/automation_server/github_server.py

```diff
@@ -0,0 +1,58 @@
+from ..error_state import HasErrorState
+from ...lib.module_arguments import ModuleArgumentParser
+from .base_server import BaseServerForHostingBuild, BaseServerForTrigger
+
+__all__ = [
+    "GithubServer",
+]
+
+
+class GithubServer(BaseServerForHostingBuild, BaseServerForTrigger, HasErrorState):
+    @staticmethod
+    def define_arguments(argument_parser: ModuleArgumentParser) -> None:
+        parser = argument_parser.get_or_create_group("Github Actions variables",
+                                                     "Github Actions specific parameters")
+
+        parser.add_argument("--github-server", "-ghs", dest="server_url", metavar="GITHUB_SERVER_URL",
```

Comment:
I propose to set the default value to https://github.com

Reply:
We don't need this, because we use one of GA default variables:
https://docs.github.com/en/actions/learn-github-actions/environment-variables#default-environment-variables

Reply:
Also I propose to stop using multi-character parameter names for compact form (like -ghs, -ghr, -ghri).
Traditionally linux utilities use single-character compact form case sensitive flags.
We already violate this rule in multiple places, but at least we can stop making things worse.

Reply:
Our options are:
1. Use some semi-random case-sensitive letters for compact form flags
2. Drop compact form

Reply:
> We don't need this, because we use one of GA default variables:
Well, ok. Then we need to correct help message to mention that the GA sets this variable already and typically user doesn't need to set any of these parameters.

Reply:
> Also I propose to stop using multi-character parameter names for compact form (like -ghs, -ghr, -ghri). Traditionally linux utilities use single-character compact form case sensitive flags. We already violate this rule in multiple places, but at least we can stop making things worse.

Ok, I was just looking at other arguments

> Well, ok. Then we need to correct help message to mention that the GA sets this variable already and typically user doesn't need to set any of these parameters.

Ok